src/OpenTelemetry/Trace/ActivityProcessorPipelineBuilder.cs

```diff
@@ -94,8 +94,7 @@ internal ActivityProcessor Build()
             }
             else if (this.Exporter != null)
             {
-                // TODO: Make this BatchingActivityProcessor once its available.
-                exportingProcessor = new SimpleActivityProcessor(this.Exporter);
+                exportingProcessor = new BatchingActivityProcessor(this.Exporter);
```

Comment:
Don't know what should be the default. Lets keep BatchingProcessor for now. And revisit the area after beta.